src/Core/Command/UploadAvatarHandler.php

```diff
@@ -101,7 +101,7 @@ public function handle(UploadAvatar $command)
             $manager = new ImageManager;
 
             // Explicitly tell Intervention to encode the image as JSON (instead of having to guess from the extension)
-            $encodedImage = $manager->make($tmpFile)->orientate()->fit(100, 100)->encode('jpg', 100);
+            $encodedImage = $manager->make($tmpFile)->fit(100, 100)->encode('png', 100);
```

Comment:
Can you fix the comment in the line above? That says JSON and probably meant JPEG. Plus now it's PNG ;)